package.json

```diff
@@ -1278,7 +1278,7 @@
     "pretest": "npm run compile-tests",
     "test": "vscode-test",
     "test-ci": "docker/test-ci.sh",
```

Comment:
Does this need to be updated to `docker/test-ci.sh ci` so we can still do `npm run test-ci 6.0`?

Reply:
ya sure